docs/source/getting-started.md

```diff
@@ -44,16 +44,17 @@ iwr 'https://rover.apollo.dev/win/v0.17.2' | iex
 
 Rover is distributed on npm for integration with your JavaScript projects.
 
-#### Installation from NPM mirror
+#### Installing from a binary mirror
 
-If you are behind a proxy and install dependencies through a mirror or proxy of npm, add the following to your global or local `.npmrc`:
+Internally, the `npm` installer downloads router binaries from `https://rover.apollo.dev`. If this URL is unavailable, for example, in a private network, you can point the `npm` installer at another URL in one of two ways:
```

Comment:
Looks like we mention npm both as "npm" and `npm` throughout the docs. I don't have a strong preference, but will standardize in a separate PR.